### DISCUSSION TITLE
Extension manager grayed out

### DISCUSSION THREAD

**Discussion by randomhammer** [pawelsalawa/sqlitestudio#3971] [Q&A]
I am trying to load an extension into SQLIteStudio 3.2.1 on Windows 10. When I open the Extensions Manager window all options except for the blue information icon are grey. I cannot do anything in that window. Is there some prerequisite for using extensions that I have not satisfied? I have been googling this for about 30 minutes and can't find an answer.

**Comment by pawelsalawa** [comment]
Start with adding a new entry:
![image](https://user-images.githubusercontent.com/481846/107672541-4d6f5180-6c95-11eb-9a39-60dedb063efd.png)

You could have missed it, because the icon looks very similar when it's in the enabled and disabled state.
